### ISSUE TITLE
Discussion: Publicly document a11y launch/review process

### ISSUE THREAD

**Issue by pbakaus** [ampproject/wg-components#24]
Hey team,

based on the latest observations by for example @beep showing the a11y shortcomings of AMP Stories' accessibility, we decided on Friday to prioritize fixing, and take another look at improving the launch/review process for all new components and features.

This is a tracking issue to make sure we publicly document what we do in regards to a11y on this repository (and optionally on amp.dev).

Over to @nainar, and @newmuis!

Thanks,
Paul

**Comment by tobie** [comment]
Also see the [related conversation](https://github.com/ampproject/meta-ac/blob/master/meetings/2019-07-01.md#accessibility-discussion) with the @ampproject/ac.

**Comment by nainar** [comment]
Per the conversation highlighted by @tobie, I am working on a document that highlights the new process that Working Groups should follow when going from idea to launch. Hoping to share this on this issue soon. (Tentative timeline End of week?)

**Comment by nainar** [comment]
Hi folks, 

Here is the proposed outline of how we are going to be Open Sourcing the UI team's work: https://docs.google.com/document/d/1bF5tHbm-ObyjKLfoiIEAI5V-YvMsA3Bchsux5u4cKZM/edit?usp=sharing

Feel free to comment with your thoughts, concerns. Excited to get this process underway!

**Comment by LJWatson** [comment]
It's really good to see this taking shape @nainar . A couple of initial thoughts...

User needs, user stories, and user acceptance criteria need to feature 
prominently in this process. Without these things, features will be 
developed or redeveloped, with no clear evidence that they're really 
needed, and no way to assess whether they've fulfilled their intended 
purpose or not.

If a WCAG Level A or Level AA failure is identified, it must be 
considered a launch blocker bug. This needs to be communicated very 
clearly to the community.

Does the Github repository have any notion of an accessibility team? The 
WHATWG has teams of specialists, including one for accessibility, that 
it can easily @mention to draw particular issues to the attention of the 
relevant people. These are not necessarily active or regular 
participants, but people with sufficient knowledge and interest in a 
particular area.

**Comment by tobie** [comment]
@beep wrote:

> I’m curious how post-launch remediation would work.

My understanding is this would be through the normal issue tracker. @nainar can you confirm? @beep would that be acceptable from your perspective?

Related, we discussed organizing an audit for existing components during the AC's F2F meeting in London ([minutes](https://github.com/ampproject/meta-ac/blob/master/meetings/2019-03-f2f.md#accessibility)) and during the WG-UI-and-A11Y/AC meeting around accessibility ([minutes](https://github.com/ampproject/meta-ac/blob/master/meetings/2019-07-01.md)). We should continue that conversation once the work @nainar has been doing around processes has landed, for example during the contributor summit this fall.
